### ISSUE TITLE
Progress bar gets cropped to original size when using larger player

### ISSUE THREAD

**Issue by outadoc** [klemens/ff-youtube-all-html5#49]
With the latest stable *and* beta versions of the add-on, the progress bar gets cropped to whatever the original size was on the HTML5 player. It seems to reset itself every second, too.

![screenshot from 2015-03-30 20 25 28](https://cloud.githubusercontent.com/assets/490651/6903565/69f952b2-d71b-11e4-861b-44a75ee46dff.png)

Tested with every video loading method available.

**Comment by klemens** [comment]
This is a known bug, which I already tried to fix some tome ago (1a90113c92a0f6751fbea81a03d3dc1acf649188). But youtube removed the ability to fix this properly. (24129f79b7213d252fc8f8817047bbef19777f02, 8a0e787d61aa1cd61d625da97c45ccbea051a43b)

I have experimented with some possibilities to fix the display of the progress bar, but this turns out to be quite difficult. While the display itself could be fixed relatively easy, the problem is that this does not fix the player code that calculates the video position when you click on the bar. So clicking positions and displayed position would no longer be in sync.
I have tried to "hack" the code and override the sizes, but had no success, so I decided the keep it the way it is. At least it's consistent.

**Comment by outadoc** [comment]
That sucks. I guess it's still better than nothing, I hope it'll be fixable in the future somehow.

**Comment by ZatsuneNoMokou** [comment]
Youtube Center (https://github.com/YePpHa/YouTubeCenter) is able to change player size without this problem, if it can help you.

**Comment by klemens** [comment]
YTC has had a fix right from the beginning, when the new player with programmatically calculated sizes was introduced: https://github.com/YePpHa/YouTubeCenter/issues/1125
But it was very ugly, so I didn't want to include it. (see the already mentioned 8a0e787 for more details)

However I just noticed after looking at the YTC code, that this was obsoleted in 2015-05 when youtube introduced the new transparent player which is still used today. This also enabled the use of a much simpler workaround: https://github.com/YePpHa/YouTubeCenter/issues/1634

I just tested this workaround (disabling the browser-internal function `matchMedia`) and it seems to work fine, so this will probably be in the next update! Thanky for your reminder. :wink:
